### PR TITLE
Added missing categories for k8s queries

### PR DESCRIPTION
**Pull request by ruigomescx**
Added missing categories for k8s queries and general refactor of some names

### CODE REVIEW

File: assets/queries/k8s/default_service_account_binding/metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "id": "Default_Service_Account_Binding",
   "queryName": "Role Binding To Default Service Account",
   "severity": "HIGH",
-  "category": null,
+  "category": "Identity and Access Mangement",
```
Comment (rogeriopeixotocx) [inline assets/queries/k8s/default_service_account_binding/metadata.json:5]: typo "Management"

File: assets/queries/k8s/default_service_account_in_use/metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "id": "default_service_account_in_use",
   "queryName": "Default Service Account In Use",
   "severity": "MEDIUM",
-  "category": null,
+  "category": "Identity and Access Mangement",
```
Comment (rogeriopeixotocx) [inline assets/queries/k8s/default_service_account_in_use/metadata.json:5]: typo "Management"

File: assets/queries/k8s/service_account_name_undefined_or_empty/metadata.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "id": "Service_Account_Name_Undefined_Or_Empty",
   "queryName": "Service Account Name Undefined Or Empty",
   "severity": "MEDIUM",
-  "category": null,
+  "category": "Identity and Access Mangement",
```
Comment (rogeriopeixotocx) [inline assets/queries/k8s/service_account_name_undefined_or_empty/metadata.json:5]: typo "Management"

File: assets/queries/k8s/limit_capabilities_from_the_container/metadata.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
   "id": "limit_capabilities_of_the_container",
-  "queryName": "Limit Capabilities Of The Container",
+  "queryName": "Not Limited Capabilities For Container",
```
Comment (rogeriopeixotocx) [inline assets/queries/k8s/limit_capabilities_from_the_container/metadata.json:3]: Unlimited?

File: assets/queries/k8s/limit_capabilities_from_the_container/test/positive_expected_result.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [
 	{
-		"queryName": "Limit Capabilities Of The Container",
+		"queryName": "Not Limited Capabilities For Container",
```
Comment (rogeriopeixotocx) [inline assets/queries/k8s/limit_capabilities_from_the_container/test/positive_expected_result.json:3]: Unlimited?

File: assets/queries/k8s/limit_capabilities_of_the_psp/metadata.json [hunk 6]
```diff
@@ -1,8 +1,8 @@
 {
   "id": "limit_the_capabilities_from_the_psp",
-  "queryName": "Limit The Capabilities From The PSP",
+  "queryName": "Not Limited Capabilities For Pod Security Policy",
```
Comment (rogeriopeixotocx) [inline assets/queries/k8s/limit_capabilities_of_the_psp/metadata.json:3]: Unlimited?

File: assets/queries/k8s/limit_capabilities_of_the_psp/test/positive_expected_result.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 [
 	{
-		"queryName": "Limit The Capabilities From The PSP",
+		"queryName": "Not Limited Capabilities For Pod Security Policy",
```
Comment (rogeriopeixotocx) [inline assets/queries/k8s/limit_capabilities_of_the_psp/test/positive_expected_result.json:3]: Unlimited?

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/default_service_account_binding/metadata.json (hunk 1 above)
Comment (ruigomescx) [reply assets/queries/k8s/default_service_account_binding/metadata.json]: fixed

File: assets/queries/k8s/default_service_account_in_use/metadata.json (hunk 2 above)
Comment (ruigomescx) [reply assets/queries/k8s/default_service_account_in_use/metadata.json]: fixed

File: assets/queries/k8s/service_account_name_undefined_or_empty/metadata.json (hunk 3 above)
Comment (ruigomescx) [reply assets/queries/k8s/service_account_name_undefined_or_empty/metadata.json]: fixed

File: assets/queries/k8s/limit_capabilities_from_the_container/metadata.json (hunk 4 above)
Comment (ruigomescx) [reply assets/queries/k8s/limit_capabilities_from_the_container/metadata.json:3]: Prefer the use of Not Limited in this case

File: assets/queries/k8s/limit_capabilities_from_the_container/test/positive_expected_result.json (hunk 5 above)
Comment (ruigomescx) [reply assets/queries/k8s/limit_capabilities_from_the_container/test/positive_expected_result.json:3]: Prefer the use of Not Limited in this case

File: assets/queries/k8s/limit_capabilities_of_the_psp/metadata.json (hunk 6 above)
Comment (ruigomescx) [reply assets/queries/k8s/limit_capabilities_of_the_psp/metadata.json:3]: Prefer the use of Not Limited in this case

File: assets/queries/k8s/limit_capabilities_of_the_psp/test/positive_expected_result.json (hunk 7 above)
Comment (ruigomescx) [reply assets/queries/k8s/limit_capabilities_of_the_psp/test/positive_expected_result.json:3]: Prefer the use of Not Limited in this case

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**